modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_policy_management/v1/IamPolicyManagementExamples.java

```diff
@@ -1188,5 +1216,317 @@ public static void main(String[] args) throws Exception {
+      // end-replace_action_control_template
+    } catch (ServiceResponseException e) {
+        logger.error(String.format("Service returned status code %s: %s%nError details: %s",
+          e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      System.out.println("listActionControlTemplates() result:");
+      // begin-list_action_control_templates
+      ListActionControlTemplatesOptions listActionControlTemplatesOptions = new ListActionControlTemplatesOptions.Builder()
+        .accountId(exampleAccountId)
+        .build();
+
+      Response<ActionControlTemplateCollection> response = service.listActionControlTemplates(listActionControlTemplatesOptions).execute();
+      ActionControlTemplateCollection actionControlTemplateCollection = response.getResult();
+
+      System.out.println(actionControlTemplateCollection);
+      // end-list_action_control_templates
+    } catch (ServiceResponseException e) {
+        logger.error(String.format("Service returned status code %s: %s%nError details: %s",
+          e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      System.out.println("createActionControlTemplateVersion() result:");
+      // begin-create_action_control_template_version
+
+      // Construct an instance of the TemplateActionControl model
+      TemplateActionControl templateActionControlModel = new TemplateActionControl.Builder()
+        .serviceName("am-test-service")
+        .description("Test Java SDK integration testcases")
+        .actions(java.util.Arrays.asList("am-test-service.test.create"))
+        .build();
+
+      CreateActionControlTemplateVersionOptions createActionControlTemplateVersionOptions = new CreateActionControlTemplateVersionOptions.Builder()
+      .description("ActionControl SDK Test template without actionControl")
+      .actionControl(templateActionControlModel)
+      .committed(true)
+      .actionControlTemplateId(exampleActionControlTemplateId)
+      .build();
+
+      // Invoke operation
+      Response<ActionControlTemplate> response = service.createActionControlTemplateVersion(createActionControlTemplateVersionOptions).execute();
+      ActionControlTemplate actionControlTemplate = response.getResult();
+      exampleActionControlTemplateVersion = actionControlTemplate.getVersion();
+      System.out.println(actionControlTemplate);
+      // end-create_action_control_template_version
+
+    } catch (ServiceResponseException e) {
+        logger.error(String.format("Service returned status code %s: %s%nError details: %s",
+          e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      System.out.println("listActionControlTemplateVersions() result:");
+      // begin-list_action_control_template_versions
+      ListActionControlTemplateVersionsOptions listPolicyTemplateVersionsOptions = new ListActionControlTemplateVersionsOptions.Builder()
```

Comment:
copy/paste error in variable name?
```suggestion
      ListActionControlTemplateVersionsOptions listActionControlTemplateVersionsOptions = new ListActionControlTemplateVersionsOptions.Builder()
```

Reply:
Yes, fixing now.. Thanks